portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,30 +31,40 @@
   @Override
   public void init() {
     messages = new ArrayList<>();
-    messages.add("Placeholder message #1.");
-    messages.add("Placeholder message #2.");
-    messages.add("Placeholder message #3.");
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Construct comment from user request.
+    String comment = request.getParameter("comment-input");
+    String name = request.getParameter("name-input");
```

Comment:
small nit: make these two parameters constants declared as private static final variables

Reply:
The compiler surfaces an error when I declare these variables as private static final variables -- this might be because doPost() is a public function. It seems like `final` is the only modifier that is accepted. Is there an alternate way to declare these variables as private static final variables?